cog/api/gift.py

```
@@ -0,0 +1,64 @@
+import requests
+
+
+class Gift:
+    def __init__(self, api_key: str, guild_id: int, recipient_id: int):
```

Ruff

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)